packages/itwin/ec3-widget/src/components/Templates.tsx

```diff
@@ -54,7 +54,7 @@ export const Templates = ({ onClickCreate, onClickTemplateTitle }: TemplateProps
           displayName: x.displayName,
           description: x.description ?? "",
           id: x.id,
-          labels: x.labels,
+          labels: [],
```

Comment:
Is this intentional?

Reply:
Seems like labels isn't even being used in this component. Eventually we are going to have to switch to the provided types in `insights-client`. That's for another day.